zombies.js

```diff
@@ -91,9 +124,10 @@
  *
  * @name checkPack
  */
-
-
-/**
+ checkPack() {
+  console.log(this._pack);
```

Comment:
The intent is that you would format an output of the content of the pack, not just dump console.log. You have limited control of console.logs output as it relates to arrays.